share/wrf_timeseries.F

```diff
@@ -232,7 +235,8 @@ SUBROUTINE calc_ts_locations( grid )
                CLOSE(UNIT=iunit)
 
                ts_profile_filename = grid%ts_filename(k)
-               DO j=1,5
+               !PSH - had to change from 5 to 6 to include header in 'WW'
```

Comment:
As written, this comment isn't quite as useful as it could be (and it has initials). I haven't actually tried it, but would simply setting the loop to
```fortran
               DO j=1,SIZE(ts_file_endings)
```
and omitting the comment altogether work?

Reply:
I have incorporated this change and removed the comment.